src/compiler/crystal/crystal_path.cr

```diff
@@ -45,60 +45,69 @@ module Crystal
     end
 
     private def find_in_path_relative_to_dir(filename, relative_to)
-      if relative_to.is_a?(String)
-        # Check if it's a wildcard.
-        if filename.ends_with?("/*") || (recursive = filename.ends_with?("/**"))
-          filename_dir_index = filename.rindex('/').not_nil!
-          filename_dir = filename[0..filename_dir_index]
-          relative_dir = "#{relative_to}/#{filename_dir}"
-          if File.exists?(relative_dir)
-            files = [] of String
-            gather_dir_files(relative_dir, files, recursive)
-            return files
-          end
-        else
-          relative_filename = "#{relative_to}/#{filename}"
+      return unless relative_to.is_a?(String)
```

Comment:
I know this was in the original code, but why not have relative_to be typed to a String right in the signature?

Reply:
Good point. I will do this, I think it will simplify code (I thought it was being passed other things than `String` or `Nil` but it doesn't seem to be the case). Thanks!

Reply:
Done!

Reply:
Actually, never mind. This can be done in a separate PR. If somebody wants to do it, please go ahead.